tests/model/test_core.py

```diff
@@ -223,6 +223,20 @@ def test_observed_type(self):
         assert x1.type.dtype == X.type.dtype
         assert x2.type.dtype == X.type.dtype
 
+    def test_observed_compute_test_value(self):
+        try:
+            with pytensor.config.change_flags(compute_test_value="raise"):
+                rng = np.random.default_rng(123)
+                data = rng.standard_normal(100)
+                with pm.Model():
+                    obs = pm.Normal(
```

Comment:
should test something about the tag.test_value of obs

Reply:
I added a test that compares `shape` and `dtype` with `data`. Let me know if you had something else in mind!